test/testlib.py

```diff
@@ -6,10 +6,18 @@
 import logging
 import random
 import shutil
+import json
 
 import testutil
 import btcrpc
 import litrpc
+import requests
+
+
+from bitcoinrpc.authproxy import AuthServiceProxy, JSONRPCException
```

Comment:
The reason I didn't just use bitcoinrpc originally is because I didn't want the testing framework to depend on external libraries, *but* if it's worth it then it's fine.  You might want to remove that `btcrpc` now though.